src/opm/parser/eclipse/share/keywords/000_Eclipse100/F/FOAMADS

```diff
@@ -0,0 +1,7 @@
+{"name" : "FOAMADS" , "sections" : ["SPECIAL", "PROPS"], "size" : {"keyword" : "TABDIMS" , "item" : "NTSFUN"} , "items" :
+    [
+        {"name":"DATA", "value_type":"DOUBLE","size_type":"ALL", "dimension":["FoamDensity", "1"]}
```

Comment:
Is the string `"FoamDensity"` defined in the unit system parsing code?

Reply:
I assume the changes in src/opm/parser/eclipse/Units/UnitSystem.cpp are what you requested?